msteams-platform/toolkit/TeamsFx-SDK.md

```diff
@@ -10,25 +10,28 @@ ms.date: 11/29/2021
 
 # TeamsFx SDK
 
-TeamsFx helps to reduce the developer tasks by using Teams SSO and accessing cloud resources down to single line statements with zero configuration. TeamsFx SDK is built to be used in browser and Node.js environment, common scenarios include:
+TeamsFx helps to reduce your tasks by using Microsoft Teams Single-Sign-On (Teams SSO) and accessing cloud resources down to single line statements with zero configuration. You can use TeamsFx SDK in browser and Node.js environment. TeamsFx core functionalities can be accessed in client as well as server environment and you can write user authentication code in a simplified way for:
 
-* Teams tab application
-* Azure Function
+* Teams tab
 * Teams bot
-
-You can use the TeamsFx SDK to:
-
-* Access the core functionalities in client and server environment.
-* Write user authentication code in a simplified way.
+* Azure function
 
```

Comment:
It should be `Azure Functions` as mentioned in its doc: https://docs.microsoft.com/en-us/azure/azure-functions/functions-overview

Reply:
This comment has been implemented.